include/conversation.php

```diff
@@ -727,7 +727,12 @@ function conversation_fetch_comments($thread_items, $pinned) {
 			&& ($row['thr-parent'] == $row['parent-uri']) && ($row['received'] > $received)
 			&& Contact::isSharing($row['author-id'], $row['uid'])) {
 			$direction = ['direction' => 3, 'title' => DI::l10n()->t('%s reshared this.', $row['author-name'])];
-			$actor = ['link' => $row['author-link'], 'avatar' => $row['author-avatar'], 'name' => $row['author-name']];
+
+			$author = ['uid' => 0, 'id' => $row['author-id'],
+				'network' => $row['author-network'], 'url' => $row['author-link']];
+			$url = '<a href="'. Contact::magicLinkByContact($author) .'">' . htmlentities($row['author-name']) . '</a>';
```

Comment:
The href value must be HTML encoded as well.

Reply:
Are you sure with this?

Reply:
Yes, since we will add this HTML snippet as is to the template without relying on Smarty encoding.

Reply:
But when we encode the URL then the URL will be invalid?

Reply:
HTML encode, not URL encode.

Reply:
Okay, changed.